src/3rdparty/mjcpy2/CMakeLists.txt

```diff
@@ -46,4 +47,4 @@ message("osg includes: ${OSG_INCLUDE_DIR}")
 include_directories(SYSTEM "${CMAKE_SOURCE_DIR}")
 include_directories(SYSTEM "${CMAKE_SOURCE_DIR}/Boost.NumPy")
 boost_python_module(mjcpy mjcpy2.cpp  mujoco_osg_viewer.cpp)
-target_link_libraries(mjcpy "${CMAKE_SOURCE_DIR}/mjpro/libmujoco.so" ${OSG_LIBRARIES} boost_numpy)
+target_link_libraries(mjcpy "${CMAKE_SOURCE_DIR}/mjpro/bin/libmujoco131.so" ${OSG_LIBRARIES} boost_numpy)
```

Comment:
Can we add support for cmake flag to use the old version? Then we can phase it out once everyone switches over. Probably fine to default to the new version.